Executables/EventHandlerService/test/EventHandlerService.Tests/EventHandlerService.Tests.csproj

```diff
@@ -24,8 +24,8 @@
     </ItemGroup>
 
     <ItemGroup>
-        <ProjectReference Include="..\..\..\BuildingBlocks\src\BuildingBlocks.Infrastructure\BuildingBlocks.Infrastructure.csproj" />
-        <ProjectReference Include="..\..\..\BuildingBlocks\src\UnitTestTools\UnitTestTools.csproj" />
+        <ProjectReference Include="..\..\..\..\BuildingBlocks\src\BuildingBlocks.API\BuildingBlocks.API.csproj" />
```

Comment:
What's this change for?

Reply:
This is a mistake on my side, the strange thing I really don't understand how did this change happened. Reverted to original reference.